src/main/resources/app.conf

```diff
@@ -80,35 +80,46 @@ app {
       # the initial delay before retrying that will be multiplied with the 'factor' for each attempt
       initial-delay = 100 milliseconds
       initial-delay = ${?KEY_VALUE_STORE_RETRY_INITIAL_DELAY}
+      # the maximum delay applied to the retrying strategy
+      max-delay = 10 minutes
       # maximum number of retries in case of failure (applicable only for strategy "exponential")
       max-retries = 7
       max-retries = ${?KEY_VALUE_STORE_RETRY_MAX_RETRY}
-      # the exponential factor (applicable only for strategy "exponential")
-      factor = 2
+      # the exponential random factor (applicable only for strategy "exponential")
+      factor = 0.2
       factor = ${?KEY_VALUE_STORE_RETRY_FACTOR}
```

Comment:
rename env override: `KEY_VALUE_STORE_RETRY_RANDOM_FACTOR` to be the same as in IAM